scroll-into-view/index.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+let $ = require('jquery');
+
+/**
+ * @ngdoc directive
+ * @name buildium.common.bdScrollIntoView
+ *
+ * @restrict 'A'
+ *
+ * @description
+ *
+ * If this directive's property evalutes to true, the element will be scrolled into view.
+ *
+ * The speed can be set with the attribute `bd-scroll-into-view-speed`
+ *
+ * @depends buildium.common.services.page
+ *
+ * <!-- example #1 -->
+ *    <input bd-scroll-into-view="true" />
+ * <!-- /example -->
+ *
+ * <!-- example #2: a very sloooow scroll -->
+ *    <input bd-scroll-into-view="vm.someVariable" bd-scroll-into-view-speed="1500" />
+ * <!-- /example -->
+ */
+
+function scrollToElement(element, fn, duration) {
+    let target = element.length ? element : $(element);
+    let callback = (fn && typeof fn === 'function') ? fn : function emptyFn() {};
```

Comment:
just the `typeof` check would be enough for the condition here right? since a function is truthy